query/security/ldap/src/test/java/org/codice/ddf/admin/ldap/TestLdapServer.java

```diff
@@ -86,13 +86,13 @@ public static String getBasicAuthDn() {
 
     public static int getLdapPort() {
         // return server.getListenPort("ldap");
-        return 1389;
+        return 11389;
```

Comment:
There's currently a pr up for moving the DynamicPort https://github.com/codice/ddf/pull/2090/files. We should fix this as soon as it is merged.